src/components/Card/__tests__/Card.test.js

```diff
@@ -0,0 +1,57 @@
+import '@testing-library/jest-dom/extend-expect'
+import { render } from '@testing-library/svelte'
+import Card from '../Card.svelte'
+
+describe('Card', () => {
+    describe('When the card is rendered with close option selected', () => {
+        it('should render the card with the correct values and green close option', () => {
+            const { getByText } = render(Card, {
+                item: { symbol: 'ABCD', close: [1, 2, 3], open: [4, 5, 6] },
+                color: 'purple',
+                options: [
+                    {
+                        label: 'Close',
+                        value: 'close',
+                    },
+                    {
+                        label: 'Open',
+                        value: 'open',
+                    },
+                ],
+                selected: 'close',
+                current: 1,
+            })
+
+            expect(getByText('$ABCD')).toBeInTheDocument()
+            expect(getByText('Close (selected)')).toBeInTheDocument()
+            expect(getByText('2')).toBeInTheDocument()
```

Comment:
What is this checking?

Reply:
Okay correct me if I am wrong - this is checking the close value based on the 'current' index...

Reply:
yep